Assets/MixedRealityToolkit/Providers/UnityInput/GenericJoystickController.cs

```diff
@@ -145,25 +141,6 @@ protected void UpdateSingleAxisData(MixedRealityInteractionMapping interactionMa
                         // Raise input system Event if it enabled
                         InputSystem?.RaiseFloatInputChanged(InputSource, ControllerHandedness, interactionMapping.MixedRealityInputAction, interactionMapping.FloatData);
                     }
-                    return;
-                default:
-                    Debug.LogWarning("Unhandled Interaction");
-                    return;
-            }
-
-            // If our value changed raise it.
-            if (interactionMapping.Changed)
```

Comment:
Why are we removing this section? It seems to be raising OnInputDown and Up events. Is it handled somewhere else?

Reply:
Not removed, just moved to the Button handler.
Did various tests in both cases and button is the better fit and handles more scenarios.  Will be validating and making the same change in the XRTK following this testing.